data/net/minecraft/CharPredicate.mapping

```diff
@@ -0,0 +1,13 @@
+CLASS net/minecraft/CharPredicate
+	METHOD and (Lnet/minecraft/CharPredicate;)Lnet/minecraft/CharPredicate;
+		ARG 1 charTest
+	METHOD lambda$and$0 (Lnet/minecraft/CharPredicate;C)Z
+		ARG 2 character
+	METHOD lambda$negate$1 (C)Z
+		ARG 1 character
+	METHOD lambda$or$2 (Lnet/minecraft/CharPredicate;C)Z
+		ARG 2 character
+	METHOD or (Lnet/minecraft/CharPredicate;)Lnet/minecraft/CharPredicate;
+		ARG 1 charTest
+	METHOD test (C)Z
+		ARG 1 character
```

Comment:
I suggest the parameters be renamed to mimic [`IntPredicate`](https://docs.oracle.com/en/java/javase/16/docs/api/java.base/java/util/function/IntPredicate.html): `value` for the `test(char)` (and the lambdas), `predicate` for the `CharPredicate` parameters.